pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -636,6 +638,33 @@ func applyRedirect(out *route.Route, redirect *networking.HTTPRedirect, port int
 	out.Action = action
 }
 
+func applyDirectResponse(out *route.Route, directResponse *networking.HTTPDirectResponse, port int) {
+	action := &route.Route_DirectResponse{
+		DirectResponse: &route.DirectResponseAction{
+			Status: directResponse.Status,
+		},
+	}
+
+	if directResponse.Body != nil {
+		switch op := directResponse.Body.Specifier.(type) {
+		case *networking.HTTPBody_String_:
```

Comment:
can we change this `HTTPBody_String_`, it does not align with our naming convention

Reply:
I'm not sure what you mean, this is a protobuf generated struct, this comes from https://github.com/istio/api/blob/master/networking/v1alpha3/virtual_service.proto#L1615

I guess since the field is named `string` protobuf appends a `_` at the end which does not happen with `bytes` field